packages/cli/commands/project/cloneApp.js

```diff
@@ -81,6 +79,7 @@ exports.handler = async options => {
       });
       appId = appIdResponse.appId;
     }
+    trackCommandMetadataUsage('clone-app', { appId }, accountId);
```

Comment:
Do we need more metadata here/is there a better place to put this tracking event?